test/run/test.run.js

```diff
@@ -115,3 +116,23 @@ describe("fx-runner start", function () {
     });
   });
 });
+
+describe("concat binary arguments", function () {
+  it("binary arguments as string", function (done) {
```

Comment:
if you remove the `done` argument then you won't have to call done at the end of the test. By leaving out the argument, the test will magically become synchronous.